.travis.yml

```diff
@@ -1,3 +1,4 @@
+sudo: false

```

Comment:
This is nice but it makes the build fail in Travis. The failures may be because of this.